docs/functions/build-run-deploy-func.md

```diff
@@ -0,0 +1,45 @@
+# Building, running, or deploying functions
+
+After you have created a function project, you can build, run, or deploy your function, depending on your use case.
+
+## Running a function
+
+--8<-- "run-func-intro.md"
+
+### Prerequisites
+
+- You have a Docker daemon on your local machine. This is already provided if you have used the Quickstart installation.
+
+- You have access to a container registry and are able to push images to this registry.
```

Comment:
```suggestion
```
Users do not need access to an external container registry in order to run a function.